Mathlib/Init/Logic.lean

```diff
@@ -120,12 +100,10 @@ alias ⟨not_of_not_not_not, _⟩ := not_not_not
 -- FIXME
 -- attribute [congr] not_congr
 
-@[deprecated and_comm] theorem and_comm' (a b) : a ∧ b ↔ b ∧ a := and_comm
 #align and.comm and_comm
-#align and_comm and_comm'
+#align and_comm and_comm
 
-@[deprecated and_assoc] theorem and_assoc' (a b) : (a ∧ b) ∧ c ↔ a ∧ (b ∧ c) := and_assoc
-#align and_assoc and_assoc'
+#align and_assoc and_assoc
```

Comment:
I think these two you should keep simply because they help teach people how the deprecation system works. Certainly this is how I learned about it.

Maybe @digama0 has a different opinion, though.

Reply:
In general, I'd rather have documentation in more discoverable places (than some undiscoverable corner of mathlib). I'm not sure if such a standard place already exists, though.

Reply:
Nono, this is not about documentation. This is about the fact that ported code often contains `and_comm'` or `and_assoc'` and therefore the people porting the code are confronted with an (easy) task of undeprecation.

Reply:
We're going to need to do something special if we want specific deprecations to last past the cutoff date.

Reply:
To be fair, I'm not sure how much more code we're going to port. So maybe this is moot.

Reply:
:bulb: Could the porting wiki be a place to mention this? After all, this is where I'd look if I where to port code and want advice.

Reply:
Sure yeah, I'm happy with that!